package.json

```diff
@@ -53,11 +46,6 @@
     "@babel/preset-flow": "^7.7.4",
     "@babel/preset-react": "^7.7.4",
     "babel-jest": "^24.9.0",
-    "eslint": "^8.56.0",
```

Comment:
eslint still provides plenty of value and I'd like to keep it in this project. The configuration needs to be adjusted though so it does not conflict with prettier (i.e. disable all formatting rules) and so the things it reports on are helpful to this project.

I think it would be fine to do that in a fast followup if you prefer to land this and then bring eslint back in a good state. I'd like to have eslint back on again before the next version is published.

Reply:
Oh, I must've missed something. What are we using ESlint for other than syntax?

I can revert that commit, or we can make a separate PR after this one. Just need to understand what to keep before I start working on it.

Reply:
It looks like the .eslintrc.js config extends `eslin-config-airbnb-base` which is full of useful checks. The main ones I like are the one that prevents leaving unused variables/imports behind (no-unused-vars) and the one that prevents using undefined variables (no-undef).

Reply:
Ah, I see.

I'm feeling a little hesitant to take this on. Perhaps one of you would be up for reintroducing ESlint? Can do this in this PR if you want, or a follow-up, as you wish. Else I will try to work out some time to get reacquainted with ESlint and all its configuration possibilities.

Reply:
I can re-introduce it in a follow up.

Reply:
Great!